packages/eslint-plugin/src/rules/jsx-pragma/index.ts

```diff
@@ -66,10 +66,7 @@ const findReactDeclarationWithDefaultImport = (
 };
 
 function createFixer(context: Rule.RuleContext, source: SourceCode, options: Options) {
-  const compiledImports = findLibraryImportDeclarations(context, [
-    COMPILED_IMPORT,
-    ...options.importSources,
-  ]);
+  const compiledImports = findLibraryImportDeclarations(context, options.importSources);
```

Comment:
should we update `findLibraryImportDeclarations` too?

Reply:
Yeah, you got it, this `createFixer(…, options)` comes from:
```tsx
const options: Options = {  
  // …
  importSources: [...DEFAULT_IMPORT_SOURCES, ...(optionsRaw.importSources ?? [])],
};
```

Updated `sources = [COMPILED_IMPORT]` to be `sources = DEFAULT_IMPORT_SOURCES` in 6b49bde6

Only other remaining `COMPILED_IMPORT` used in _isolation_ is either for codemod or autofixes or for `styled` imports, which don't exist in `DEFAULT_IMPORT_SOURCES` (might be odd, but alas). 🙏

Reply:
ahh cool, yeah that makes sense